packages/client/hmi-client/src/components/data-explorer/search-bar.vue

```diff
@@ -17,6 +17,7 @@
 			></i>
 		</span>
 		<i class="pi pi-history" />
+		<i class="pi pi-image" title="Search by Example" @click="toggleSearchByExample" />
```

Comment:
This should be a `<Button />`

Reply:
this is a temp placeholder so I would no worry about it since it is actually not hooked to anything!